index.js

```diff
@@ -0,0 +1,37 @@
+
+function updateHeading(newHeading) {
+    let heading = document.querySelector("h2");
```

Comment:
Чому б не використати тут `const`? 

```suggestion
    const heading = document.querySelector("h2");
```


Також варто помітити, чим буде відрізнятися поведінка, якщо об'явити змінну за допомогою `var`.

Reply:
Цікаво, буду гуглити. Дякую.